Behaviors/GroupBoxBehaviors.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections.Generic;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Media;
+using Thingie.WPF.Converters;
+
+namespace Thingie.WPF.Behaviors
+{
+    public class GroupBoxBehaviors
+    {
+        public static readonly DependencyProperty HideOnChildrenNotVisibleProperty = DependencyProperty.RegisterAttached("HideOnChildrenNotVisible", typeof(bool), typeof(GroupBoxBehaviors), new PropertyMetadata(default(bool), OnHideOnChildrenNotVisibleChanged));
+        public static void SetHideOnChildrenNotVisible(GroupBox element, bool value) { element.SetValue(HideOnChildrenNotVisibleProperty, value); }
+        public static bool GetHideOnChildrenNotVisible(GroupBox element) { return (bool)element.GetValue(HideOnChildrenNotVisibleProperty); }
+
+        private static void OnHideOnChildrenNotVisibleChanged(DependencyObject obj, DependencyPropertyChangedEventArgs args)
+        {
+            GroupBox groupBox = obj as GroupBox;
+
+            if (groupBox == null)
+                return;
+
+            if (groupBox.IsLoaded)
+            {
+                var multiBinding = new MultiBinding
```

Comment:
Is the code in the "if" part different than the code in the event handler in the "else" part?

Reply:
Yeah, it's not different. I wanted to extract this to a private method but my brain wasn't working when trying to decide on a method name since the method would either create or remove a binding. Any suggestions?

Reply:
SetupVisibilityBinding